core/schema/src/main/liquibase/foundation-2017/changelog.xml

```diff
@@ -39,6 +39,22 @@
     </rollback>
   </changeSet>
 
+	<!-- NMS-9559 store forwarders -->
```

Comment:
These changes should be in the changelog for foundation-2018 (a new one may need to be created)

Reply:
yup I'm going to do the changes